.circleci/config.yml

```diff
@@ -4,9 +4,15 @@ executors:
   php-5_6:
     docker:
       - image: circleci/php:5.6
```

Comment:
@ismaelmedinapacklink could you update this legacry image for the new one? https://circleci.com/docs/2.0/circleci-images/